cmake/SearchForStuff.cmake

```diff
@@ -290,4 +290,3 @@ if(NOT USE_SYSTEM_YAML)
 endif()
 
 add_subdirectory(3rdparty/libchdr/libchdr EXCLUDE_FROM_ALL)
-include_directories(3rdparty/libchdr/libchdr/include)
```

Comment:
Wouldn't this break the search for it as a submodule if we wanted to use the local copy?

Reply:
No more than every other place I deleted an `include_directories`